src/main/java/CarRacing/Racing.java

```diff
@@ -0,0 +1,46 @@
+package CarRacing;
+
+import CarRacing.view.InputView;
+import CarRacing.view.ResultView;
+
+import java.util.Random;
+import java.util.*;
+
+public class Racing {
+
+    static InputView inputView = new InputView();
+    static ResultView resultView = new ResultView();
+    private final Random random = new Random();
+
+    private static final int CONDITION_NUMBER = 4;
+    private static final int DISTANCE_PER_TRY = 1;
+
```

Comment:
```suggestion

    private static final int CONDITION_NUMBER = 4;
    private static final int DISTANCE_PER_TRY = 1;

    static InputView inputView = new InputView();
    static ResultView resultView = new ResultView();
    private final Random random = new Random();
```
view 멤버 변수에는 static 키워드가 있고
random 변수에는 private final 키워드가 있는데요
어떤 이유로 나눠서 선언하신 건지 궁금합니다!

추가로 [컨벤션 가이드](https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html#11684)를 보면 선언 위치를 안내하고 있어요. 참고하시고 반영해 보세요!

Reply:
원래 의도는 inputVeiw와 resultView는 커스텀 클래스라서 1번만 호출하고(오버라이딩은 허용), 
random은 많이 호환되는 기본 클래스이기에 오버라이딩 막으려고 했었어요(여러번 호출 허용). 
그런데 다시 생각해보니 불필요한 구분이었어서 전부 붙일게요